code/MDL/HalfLife/HL1MDLLoader.cpp

```diff
@@ -374,7 +374,7 @@ void HL1MDLLoader::read_texture(const Texture_HL1 *ptexture,
     for (outheight = 1; outheight < ptexture->height; outheight <<= 1)
         ;
 
-    if (outheight > BuffenLen) {
+    if (static_cast<size_t>(outheight) > BuffenLen) {
         outheight = BuffenLen;
```

Comment:
There should probably be a **static_cast<int>**, otherwise this could raise a warning when building on x64.